### DISCUSSION TITLE
[BUG] Hiding climate actions only works when the control is expanded to full screen

### DISCUSSION THREAD

**Discussion by i-am-shodan** [NemesisRE/kiosk-mode#128] [Ideas]
**Describe the bug**

This PR (https://github.com/NemesisRE/kiosk-mode/pull/125) hides elements in the thermostat control. The problem is it only takes effect when the card is 'fullscreen' not when its embedded in a dashboard.

**To Reproduce**
Here are screenshots of my dashboard when ?hide_dialog_climate_actions has been used. In the first the climate control is added to the dashboard and the advanced control are visible. In the second I've clicked to make the control full screen and these have correctly been removed.

![image](https://github.com/NemesisRE/kiosk-mode/assets/6901273/405f7a7d-cb8c-4fcf-a3b2-4f87a2aa6dd1)

![image](https://github.com/NemesisRE/kiosk-mode/assets/6901273/dae9f925-0355-49cc-a78b-61ecdeb7edae)

**Expected behavior**
Advanced controls removed everywhere

**Versions:**
 - Home Assistant: 2023.10.1
 - Kiosk-Mode 4.3.0

**Comment by NemesisRE** [comment]
This is expected cause the setting should only effect the the more info dialog. we do not change card behavior with kiosk mode.

**Comment by i-am-shodan** [comment]
That is a shame, any particular reason why this is the case? I'm trying to use my dashboard in a kiosk-like mode.

**Comment by NemesisRE** [comment]
It's because you can change the cards you use but not the more info dialog behind it. Kiosk mode is only meant to change things which are not changeable by the admin anyway.

**Comment by NemesisRE** [comment]
for card changes there is an other plugin [Card Mod by Thomas Lovén](https://github.com/thomasloven/lovelace-card-mod)

**Reply by NemesisRE** [reply]
@elchininet @Mariusthvdb
maybe we could add a collection of card mod snippets that compliment kiosk mode

probably like so
docs/card-mod/*.md

**Reply by elchininet** [reply]
Coould be a good addition, @Mariusthvdb has a big collection of those.

**Comment by elchininet** [comment]
Hi @i-am-shodan,

As @NemesisRE explained and as it is stated in the documentation:

`hide_dialog_climate_actions`: Hides all the actions **in the more-info dialogs** of climate entities. It will not affect any card that you have added to your dashboard, those ones are under your control and you can use [card-mod](https://github.com/thomasloven/lovelace-card-mod) to hide them.

Regards

**Comment by elchininet** [comment]
Hi @i-am-shodan,
[In this new document](https://github.com/NemesisRE/kiosk-mode/blob/master/KIOSK-MODE-COMPLEMENTS.md#hide-different-elements-in-native-climate-entities-home-assistant-cards) for `kiosk-mode` complements you can find a way to achieve what you were trying to do.
Regards
